services/QuillLMS/spec/queries/student_dashboard_metrics_spec.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe StudentDashboardMetrics do
+  before do
+    wednesday_noon = Time.current.beginning_of_week(:wednesday).noon
+    travel_to wednesday_noon
+  end
+
+  after do
+    travel_back
+  end
+
+  let(:today) { Date.current }
+  let(:july_second_of_this_year) { Date.parse("02-07-#{today.year}") }
+  let(:last_july_second) { today.month > 7 ? july_second_of_this_year : july_second_of_this_year - 1.year }
+
+  let(:today_timespent) { 30 }
+  let(:yesterday_timespent) { 45 }
+  let(:first_of_month_timespent) { 60 }
+  let(:july_second_timespent) { 90}
+
+  let!(:student) { create(:student) }
+  let!(:classroom) { create(:classroom) }
+  let!(:classroom_unit) { create(:classroom_unit, classroom: classroom) }
+  let!(:activity_sessions) do
+    [
```

Comment:
The four `classroom_unit: classroom_unit,` clauses can be shortened to `classroom_unit:,`

Reply:
Thank you! I am going to internalize this soon!